components/scream/data/theta.nl

```diff
@@ -0,0 +1,38 @@
+&ctl_nl
```

Comment:
That file was originally more "configurable" in homme. It contained lines like

```
rsplit            = ${HOMME_TEST_RSPLIT}
```
so that one could use cmake to cnofigure the file for the particular test.

I used this namelist for the cime-driven test. I didn't want to spend much time on hacking `buildnml`, to get this namelist configured depending on cime case parameters, so I ended up hardcoding all the values (I just wanted the exec to run, and load "reasonable" values). In the future, it should go back to contain configurable strings (like in the example above), and buildnml (or whatever cime tool is appropriate) should take care of replacing those strings with actual data depending on the compset/case.

Reply:
As for the folder in which to put those files, we could put them in a subfolder, yes. Maybe something like `cime_config/input_files` (mimicking  eam's `bld/namelist_files`), or `data/input_files` or `data/control_files`. I don't really have a preference.